package/MDAnalysis/core/universe.py

```diff
@@ -238,6 +238,26 @@ class Universe(object):
     vdwradii: dict, ``None``, default ``None``
         For use with *guess_bonds*. Supply a dict giving a vdwradii for each
         atom type which are used in guessing bonds.
+    context: str or :mod:`Guesser<MDAnalysis.guesser>`, default ``default``
```

Comment:
```suggestion
    context: str or :mod:`Guesser<MDAnalysis.guesser>`, default ``'default'``
```
just adding quotes so it's easier to quickly see it's a string.